script/DeployUsccScript.s.sol

```diff
@@ -0,0 +1,44 @@
+pragma solidity ^0.8.26;
+
+import "forge-std/Script.sol";
+import "openzeppelin-contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import "openzeppelin-contracts/proxy/transparent/ProxyAdmin.sol";
+import "src/AllowList.sol";
+import "src/USCC.sol";
+
+contract DeployScript is Script {
```

Comment:
copy pasted from existing deploy script but removed allowlist deploy since we'll be reusing the exisiting one